app/extensions/safe/ffi/ipc.js

```diff
@@ -270,7 +270,7 @@ const onSharedMDataDecision = ( e, data, isAllowed ) =>
         .catch( ( err ) =>
         {
             reqQ.req.error = err;
-            logger.error( err );
+            logger.error( 'Share MD decision error:: ', err );
             e.sender.send( 'onSharedMDataRes', reqQ.req );
             reqQ.next();
```

Comment:
Minor enhancement, we could move L274 and L275 into a `.then` statement since these two lines are common to both cases, i.e. when `authenticator.encodeMDataResp` was successful or failed., perhaps that makes it more clear what's the difference in the behavious for one case or the other.